stories/adventureco/the cat/start.txt

```diff
@@ -0,0 +1,6 @@
+[Text]
+A cute cat comes up to you and `meows`
+
+[Options]
+Pet the cat => pet_the_cat.txt
```

Comment:
Spelling error, we have `pet the cat` page link, but the page is called `pat the cat`